lib/postgres.dart

```diff
@@ -41,6 +43,24 @@ class Sql {
   factory Sql(String sql, {List<Type>? types}) =
       InternalQueryDescription.direct;
 
+  /// Looks for positional parameters in [sql] and desugars them.
+  ///
+  /// This mode is very similar to the native format understood by postgres,
+  /// except that:
+  ///  1. The character for variables is customizable (postgres will always use
+  ///     `@`). This method also uses `@` by default, but this can be changed.
+  ///  2. Not every variable needs to have en explicit index. When declaring a
+  ///     variable without an index, the first index higher than any previously
+  ///     seen index is used instead.
+  ///
+  /// For instance, `Sql.indexed('SELECT ?2, ?, ?1', substitution: '?')`
+  /// declares three variables (appearing in the order 2, 3, 1).
+  ///
+  /// Just like with [Sql.named], it is possible to declare an explicit type for
+  /// variables: `Sql.indexed('SELECT @1:int8')`.
```

Comment:
is this rather `SELECT ?1:int8`?

Reply:
By default we use `@` to declare variables with `Sql.indexed` as well.